### DISCUSSION TITLE
Test github Galaxy Brain

### DISCUSSION THREAD

**Discussion by HeathWang** [HeathWang/HWPanModal#127] [Q&A] [answered]
Test github Galaxy Brain

**Comment by HeathWang** [comment]
now I make it as answer.

**Reply by HeathWang** [reply]
I will see whether I can get Galaxy Brain

**Comment by jherskow** [comment, marked as answer]
If you unselect @HeathWang 's answer and give it to me, do we both get the galaxy brain? or will he lose his?

If he loses it that would not be fun and it should be returned

**Reply by HeathWang** [reply]
> If you unselect @HeathWang 's answer and give it to me, do we both get the galaxy brain? or will he lose his?
> 
> If he loses it that would not be fun and it should be returned

did it work?
